AppGenGuide.md

```diff
@@ -30,139 +33,121 @@ application to be deployed as a StarlingX App it needs to be designed so it can
 run on [Kubernetes](https://kubernetes.io/).
 
 Additionally, it needs to provide a [Helm Chart](https://helm.sh/)
-which will be managed via [Flux](https://fluxcd.io/).
+which will be managed via [Flux](https://fluxcd.io/) by StarlingX itself.
 
-## Generate the application
+## Generate the StarlingX Application package
 
-1. Clone the app-generator repository
-  
+1. Clone the app-generator repository.
    ```shell
    git clone https://github.com/Danmcaires/StarlingX-App-Generator.git
    ```
 
-   please note that the repository structure will be:
+   This is what you'll find in the root folder of the repository:
+ 
+   > _NOTE_: TODO why is there a reference to the poc-starlingx app already? Update image!
+
+   ![file-structure](BuildGuide/file-strucuture-1.png)
 
-    ![file-structure](BuildGuide/file-strucuture-1.png)
+2. Copy the [application's helm-chart(s)](https://helm.sh/docs/topics/charts/#the-chart-file-structure)
+   to the helm-chart directory.
+   
+   > _NOTE_: TODO we should get a path to the helm chart, not copy things around.
 
-2. Then copy the application helm-chart folders to the helm-chart app-generator
-   directory, for the messages poc, the helm-chart directory is already in the
-   expected place.
+3. Create a copy of the `app_manifest.yaml.template` file:
```

Comment:
so We'll have an app manifest template inside templates folder?

Reply:
@Danmcaires @tnpb-encora and I decided not to rename the `app_manifest.yaml` and completely remove the step that requires a copy of it.